Base/usr/share/man/man1/drain.md

```diff
@@ -0,0 +1,38 @@
+## Name
+
+drain - Print file to stdout, while progressively deleting read segments
+
+## Synopsis
+
+```**sh
+$ drain [file]
+```
+
+## Description
+
+drain is a utility for progressively deleting a file while reading segments from it.
+It is useful for example on low memory storage situations, when completing a task on
+a big file can be done when you actually need to delete it, so this utility reads the
+file and delete it in parts, and when utility reads & delete, the utility streams the
+data to another program.
+
+## Options
+
+* `-b`, `--block-size`: Base block size [in KiB] to be used during the utility operation, default is 256 KiB
+
+## Arguments
+
+* `file`: File to be read
+
+## Warning
+
+Because this utility modifies the input file to reverse it and then
+progressively prunes it, it is necessarily unsafe to be interrupted.
+If something fails in the middle, the input file is likely not recoverable.
```

Comment:
This is sort of worded as if the user would have already known that the file is reversed in-place (mostly because the sentence begins with "Because..."). So let's drop that and reword this a little bit:

> In order to progressively delete the file, `drain` reverses the file in-place to truncate blocks after they are read. Thus, it is implicitly unsafe to interrupt the utility. If the operation is interrupted, or otherwise fails, the input file is unlikely to be recoverable.